.ci/integration_test.sh

```diff
@@ -0,0 +1,240 @@
+# !/bin/bash
+
+set -e
+
+macOS=
+GOPATH_BIN=$(go env GOPATH)/bin
+SHELL_FOLDER=$(
+    cd $(dirname $0)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5569463)